opencensus/trace/ext/utils.py

```diff
@@ -63,3 +65,31 @@ def disable_tracing_url(url, blacklist_paths=None):
             return True
 
     return False
+
+
+def disable_tracing_hostname(url, blacklist_hostnames=None):
```

Comment:
Can this be combined with the `disable_tracing_url` function in this file?

Reply:
I am not sure. They both disable tracing but one is working on paths (blacklist path starting with an element from the list) and the other one with hostname (blacklist hostname/port exactly matching) with different default value. I think it's nice to have them separated for readability.

Reply:
@liyanhui1228 Do you stand on your position on this?

Reply:
It's fine to separate them :)